adagrams/game.py

```diff
@@ -1,11 +1,71 @@
+import random
+
+
 def draw_letters():
-    pass
+    letters = (["A"] * 9 + ["B"] * 2 + ["C"] * 2 + ["D"] * 4 + ["E"] * 12
+               + ["F"] * 2 + ["G"] * 3 + ["H"] * 2 + ["I"] * 9 + ["J"]
+               + ["K"] + ["L"] * 4 + ["M"] * 2 + ["N"] * 6 + ["O"] * 8
+               + ["P"] * 2 + ["Q"] + ["R"] * 6 + ["S"] * 4 + ["T"] * 6
+               + ["U"] * 4 + ["V"] * 2 + ["W"] * 2 + ["X"] + ["Y"] * 2
+               + ["Z"])
```

Comment:
Very creative! There's going to be overhead no matter what, whether we make a list or dictionary, likely we won't see it like this. Consider grabbing the `LETTER_POOL` constant variable in `tests` and use a for loop to create this list.

But, either way, good job coming up with a realistic distribution of letters to pull from!